studio/app/common/core/workflow/workflow_runner.py

```diff
@@ -122,6 +124,7 @@ def rulefile(self):
                     unique_id=self.unique_id,
                     node=node,
                     edgeDict=self.edgeDict,
+                    nwbfile=nwbfile,
```

Comment:
do not add nwbfile to algo type nodes in `snakemake.yaml`.
only `experiment.yaml` required.